### DISCUSSION TITLE
demo or sample code for open local zip file

### DISCUSSION THREAD

**Discussion by devops724** [101arrowz/fflate#144] [Q&A]
Hi there
i can save some folder as zip project in local using fflate documentations 
i try make open project i saved in local in zip format
but i can't find any guide about this
how can i open zip file in local and access it files contents, there is some files and folder in this zip file that i need read them and load in browser

**Comment by 101arrowz** [comment]
I'm not sure what exactly you're asking, could you clarify the question?

**Reply by devops724** [reply]
i found solution for open zip file 
here is sample code
```
let reader = new FileReader();
reader.onload = function (e) {
let arrayBuffer = fflate.unzipSync(new Uint8Array(reader.result));
   console.olg(JSON.parse(String.fromCharCode.apply(null, arrayBuffer["data.json"])));
}
reader.readAsArrayBuffer(file);
```
